libs/core/src/lib/dynamic-page/dynamic-page-header/header/dynamic-page-header.component.html

```diff
@@ -14,7 +14,7 @@
             </ng-container>
 
             <div class="fd-dynamic-page__title-subtitle-container">
-                <div class="fd-dynamic-page__title-container">
+                <div class="fd-dynamic-page__title-container" role="heading" aria-level="2">
```

Comment:
Just a question: why not `h2` directly, but a div with role and aria-level?

Reply:
This was Nikolay's suggestion

Reply:
He suggested: `you need to use either a heading htmlo element or role heading with aria-level`. role heading and aria-level are used when for some reason you can't use h1, h2, etc. 
<img width="780" alt="Screenshot 2024-01-25 at 9 58 50 AM" src="https://github.com/SAP/fundamental-ngx/assets/39598672/3b24518b-bee6-4e7e-bc76-733626b3963c">


https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role